versions/1/data_stream/spec.yml

```diff
@@ -3,7 +3,7 @@ spec:
   contents:
   - description: Folder containing a single data stream definition
     type: folder
-    pattern: '([a-z0-9]{2}|[a-z0-9][a-z0-9_]+[a-z0-9])'
+    pattern: '^([a-z0-9]{2}|[a-z0-9][a-z0-9_]+[a-z0-9])$'
```

Comment:
Just want to double check this pattern here. It says we allow underscores in data stream folder names, but we don't allow hyphens. Just want to make sure this is what we want.

Reply:
Yes, it's requirement enforced by Kibana, which uses hyphens to build indices.